mappings/net/minecraft/structure/pool/SinglePoolElement.mapping

```diff
@@ -17,5 +20,8 @@ CLASS net/minecraft/class_3781 net/minecraft/structure/pool/SinglePoolElement
 		ARG 3 keepJigsaws
 	METHOD method_27233 getStructure (Lnet/minecraft/class_3485;)Lnet/minecraft/class_3499;
 		ARG 1 structureManager
+	METHOD method_28877 locationCodec (Lcom/mojang/datafixers/util/Either;Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
```

Comment:
Is this a getter or creator or something else?

Reply:
creator

Reply:
Since this method is passed as an `Encoder<Either<Identifier, Structure>>` to `Codec.of`,
```suggestion
	METHOD method_28877 encodeLocation (Lcom/mojang/datafixers/util/Either;Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
```